### DISCUSSION TITLE
Domotizzare telecomando

### DISCUSSION THREAD

**Discussion by nilloi** [indomus/forum#2106] [GENERALE - Discussioni generiche]
Buonasera a tutti,
Sistemi per clonare e domotizzare  un telecomando ( 2 o 3 pulsanti) RF con Rolling Code  ( stile Broadlink ma non quello che non è compatibile) anche artigianali ( anche con  shelly etc...) per utilizzarlo da app e/o Alexa?

**Comment by 50494554524F** [comment]
il rolling code serve proprio per non poterli clonare... vedi se ci puoi attaccare un bot che prema il pulsante

**Reply by snakuzzo** [reply]
Oppure, apri il telecomando e utilizzi un attuatore con contatto pulito (es. Shelly1)

**Comment by KC-inDomus** [comment]
> Buonasera a tutti, Sistemi per clonare e domotizzare  un telecomando ( 2 o 3 pulsanti) RF con Rolling Code  ( stile Broadlink ma non quello che non è compatibile) anche artigianali ( anche con shelly etc...) per utilizzarlo da app e/o Alexa?

https://indomus.it/progetti/integrare-in-domotica-qualsiasi-telecomando-anche-rolling-code-tramite-contatto-pulito/
